doc/devel/document.rst

```diff
@@ -399,11 +399,14 @@ expression in the Matplotlib figure. In these cases, you can use the
 
 .. _writing-docstrings:
 
-Write docstrings
-================
+Write API documentation
+=======================
 
-Most of the API documentation is written in docstrings. These are comment
-blocks in source code that explain how the code works.
+The API reference documentation describes the library interfaces, e.g. inputs, outputs,
+and expected behavior. Most of the API documentation is written in docstrings. These are
+comment blocks in source code that explain how the code works. All new or edited
```

Comment:
```suggestion
comment blocks in source code that explain how the code works. All
```
We can leave this out. Our docstrings should mostly be compliant nowadays. And for the few cases that are not, either nobody notices or somebody writes ar PR, which both is ok.